packages/api/cms-api/src/page-tree/createPageTreeResolver.ts

```diff
@@ -92,7 +90,7 @@ export function createPageTreeResolver({
         @Query(() => [PageTreeNode])
         async pageTreeNodeList(
             @Args("scope", { type: () => Scope }) scope: ScopeInterface,
-            @Args("category", { type: () => Category, nullable: true }) category: PageTreeNodeCategory,
+            @Args("category", { type: () => String, nullable: true }) category: PageTreeNodeCategory,
```

Comment:
> We should keep using a GraphQL Enum here, but change the type definition in the Admin to simply string

that's what I tried first. 
this:
<img width="701" alt="Screenshot 2022-07-06 at 11 46 32" src="https://user-images.githubusercontent.com/44967610/177528816-6ce58d3e-88f7-4fd8-a951-72efb34d6ffc.png">

leads to:
<img width="612" alt="Screenshot 2022-07-06 at 12 21 28" src="https://user-images.githubusercontent.com/44967610/177529060-fd6bcf49-79ef-4fcf-a32b-a407dd2ab935.png">

so yes I can change it, but then I'd have to make a type assertion
<img width="721" alt="Screenshot 2022-07-06 at 12 21 55" src="https://user-images.githubusercontent.com/44967610/177529148-6b90f51d-dc3d-4c2b-ac91-3f20498fbd3b.png">

but I guess it is better, because the api will validate the type then. 

> this would also prevent a breaking change in all Comet apps.

There should be no breaking change with either option, unless I am missing something. These queries and mutations are all changed in the library-code.